index.html

```diff
@@ -7442,7 +7449,16 @@ <h5>Note regarding the ARIA 1.1 <rref>none</rref> role.</h5>
 		<div class="role" id="searchbox">
 			<rdef>searchbox</rdef>
 			<div class="role-description">
-				<p>A type of textbox intended for specifying search criteria. See related <rref>textbox</rref> and <rref>search</rref>.</p>
+				<p>A type of textbox intended for specifying search criteria. See related <rref>search</rref> and <rref>combobox</rref>.</p>
```

Comment:
I think this line doesn't need to be change, it makes sense to link to textbox and search!

Reply:
can you change this line back for explain you reasoning for the change?